lms/static/scripts/frontend_apps/services/client-rpc.js

```diff
@@ -10,6 +10,16 @@ import { JWT } from '../utils/jwt';
  * @prop {string} userid
  */
 
+/**
+ * @typedef {'create'|'update'|'flag'|'delete'} AnnotationEventType
```

Comment:
Perhaps at our next FE sync we could chat about types that need to be shared across projects. Or if there's a quick and dirty way to import these types from the `client`...?

Reply:
A shared package of some description would be the canonical way to do this. The package could be generated from files in the client repo. The overhead of setting that up would probably outweigh the benefits at present. For the moment I think it would be fine just to add a comment about where the canonical types for this object are in the client repo. Another option is to have all the relevant types in one file in the client repo which can easily be re-copied into the LMS app when it changes.